### PR TITLE
i2pd 2.41.0

### PR DESCRIPTION
**Pull request by r4sas**
Signed-off-by: R4SAS <r4sas@i2pmail.org>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by r4sas** [comment]
Really don't know what happend here...

**Comment by chenrui333** [comment]
> Really don't know what happend here...

test failure
```
==> /opt/homebrew/Cellar/i2pd/2.[41](https://github.com/Homebrew/homebrew-core/runs/5265589873?check_suite_focus=true#step:6:41).0/bin/i2pd --datadir=/private/tmp/i2pd-test-20220220-77950-95ffbl --pidfile=/private/tmp/i2pd-test-20220220-77950-95ffbl/i2pd.pid --daemon
  Error: test failed
  libc++abi: terminating with uncaught exception of type std::__1::system_error: mutex lock failed: Invalid argument
```

**Comment by r4sas** [comment]
Did something changed in C++ abi in last 3 months?

**Comment by r4sas** [comment]
@chenrui333 @cho-m now it works correctly.

**Comment by r4sas** [comment]
bump.

File: Formula/i2pd.rb [hunk 1]
```diff
@@ -69,3 +71,77 @@ def post_install
     Process.kill "TERM", pid
   end
 end
+
+__END__
+diff --git a/libi2pd/RouterInfo.cpp b/libi2pd/RouterInfo.cpp
+index 8d6c7120..ee7acd73 100644
+--- a/libi2pd/RouterInfo.cpp
++++ b/libi2pd/RouterInfo.cpp
+@@ -46,7 +46,7 @@ namespace data
```
Comment (SMillerDev) [inline Formula/i2pd.rb:80]: Is this submitted upstream?

Comment (r4sas) [reply Formula/i2pd.rb:80]: Yes, read commit description - I provided commit hash in upstream.

Comment (SMillerDev) [reply Formula/i2pd.rb:80]: Can't we apply that commit directly?

Comment (r4sas) [reply Formula/i2pd.rb:80]: It was created ahead of trunk code, which contains changes in modified files.

Comment (SMillerDev) [reply Formula/i2pd.rb:80]: If it's just a warning it's fine, a commit reference is much easier to keep track of than an inline patch.

**Review by SMillerDev — APPROVED** [review]
Thanks @r4sas  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**